packages/mcdoc/src/binder/index.ts

```diff
@@ -247,10 +247,15 @@ function hoist(node: ModuleNode, ctx: McdocBinderContext): void {
 		}
 
 		// hoistUseStatement associates the AST node with the binding definition in the file symbol table,
-		// which may get overridden by bindUseStatement in the later stage as an reference to the imported symbol in the global symbol table.
+		// which will get overridden by bindUseStatement in the later stage as an reference to the imported symbol in the global symbol table.
 		// This way when the user tries to go to definition on the path in the use statement,
 		// they will go to the definition in the imported file.
 
+		/**
+		 * String representation of the absolute path of the imported symbol.
+		 */
+		const importedPathString = pathArrayToString(resolvePath(path, ctx))
```

Comment:
(nitpick)

this fits comfortably on one line if you're open to that stylistically, and still renders as JSDoc

```suggestion
		/** String representation of the absolute path of the imported symbol. */
		const importedPathString = pathArrayToString(resolvePath(path, ctx))
```

Reply:
it makes more sense to consistently have JSDoc formatted this way though, and having a line break in-between is nice for readability, since the comment color is so dark

Reply:
Yeah, I think I also prefer to have it always multiline.

Reply:
works for me!